client/state/jetpack-connect/actions.js

```diff
@@ -313,29 +313,22 @@ export function authorize( queryObject ) {
 				} );
 				// Update the user now that we are fully connected.
 				userFactory().fetch();
-				return wpcom.me().sites( {
-					site_visibility: 'all',
-					include_domain_only: true,
-					fields:
-						'ID,URL,name,capabilities,jetpack,visible,is_private,is_vip,icon,plan,jetpack_modules,single_user_site,is_multisite,options', //eslint-disable-line max-len
```

Comment:
With this change, these fields are missing from this PR with respect to current behavior:

- `is_multisite`
- `is_private`
- `is_vip`
- `single_user_site`

And `meta` is included, where it wasn't before.